### ISSUE TITLE
[BUG] Skip Link Component Fails Accessibility Contrast Checker in Dark Mode

### ISSUE THREAD

**Issue by UKHOGreg** [UKHO/admiralty-design-system#526]
**Describe the bug**
Compare the foreground and background colours of the Skip Link component in Dark Mode and it fails AA contrast check.

**To Reproduce**
[Contrast Checker](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=FBE86C)

**Expected behavior**
Passes at least AA in contrast checker.

**Screenshots**
<img width="309" height="93" alt="Image" src="https://github.com/user-attachments/assets/6e6e3e66-319d-4009-a2a8-823badac807b" />

**Additional context**
